reconcile/test/test_utils_mr_clusters_updates.py

```diff
@@ -0,0 +1,84 @@
+from unittest import TestCase
+from unittest.mock import MagicMock
+
+from ruamel import yaml
+
+import reconcile.utils.mr.clusters_updates as sut
+
+from .fixtures import Fixtures
+
+fxt = Fixtures('clusters')
+
+
+class TrivialClustersUpdates(sut.CreateClustersUpdates):
```

Comment:
The approach here wasn't clear to me until I looked a little deeper into this. What's the reason behind creating this new object from `CreateClusterUpdates` versus mocking cancel, or checking for `CancelMergeRequest` to be raised? I don't think one versus the other is right or wrong, but some comments might help others to understand this decision.